model/skill.go

```diff
@@ -13,11 +16,16 @@ const (
 	DatabaseSkillType      = "database"
 )
 
-func NewSkill(id, name, skillType string) Skill {
+func NewSkill(id, name, skillType string,
```

Comment:
I think we may want to keep this the old way, and add a different constructor for `NewSkillWithLinks` and `(s Skill)AddLink` with ( link `Link`,  linkType `string` ) arguments.